src/transaction/fee.rs

```diff
@@ -127,15 +124,13 @@ fn max_of_keys(cairo_rsc: &HashMap<String, usize>, weights: &HashMap<String, f64
 }
 
 /// Calculates and charges the actual fee.
-///
```

Comment:
You shouldn't delete this line because it separates the short and full descriptions. You can check the difference if you run `cargo doc --open`